MCprep_addon/materials/skin.py

```diff
@@ -31,8 +31,17 @@
 from . import generate
 from .. import tracking
 from .. import util
+from ..spawner import spawn_util
 
 from ..conf import env
+from .prep import McprepMaterialProps
+
+
+swap_all_imgs_desc = (
+	"Swap textures in all image nodes that exist on the selected \n"
+	"material; if off, will instead seek to only replace the images of \n"
+	"nodes (not image blocks) named MCPREP_SKIN_SWAP"
+)
```

Comment:
Don't forget to use `env._` for i18n
```suggestion
swap_all_imgs_desc = (
	env._("Swap textures in all image nodes that exist on the selected \n")
	env._("material; if off, will instead seek to only replace the images of \n")
	env._("nodes (not image blocks) named MCPREP_SKIN_SWAP")
)
```

Reply:
Though I agree with the call to use env._ here, I do think we should do it only once on the final constructed multi line string block. We have no way of knowing the number of lines or breaks for other locales.